single-audit/resolve_findings/models.py

```diff
@@ -1,3 +1,82 @@
 from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
 
-# Create your models here.
+
+def validate_cfda_number(number):
+    # CFDA stands for "Catalog of Federal Domestic Assistance."
+    if len(str(number)) != 5:
+        raise ValidationError('A CFDA number must have five digits. The number \
+                               you provided is %s.' % number)
+    # @todo: Determine and add remaining kinds of CFDA number validation, when
+    #        appropriate.
+
+
+class Grantee(models.Model):
+    name = models.CharField(max_length=250)
+    # grantors = models.ManyToManyField(Agency)
```

Comment:
Is this a TODO item?

Reply:
Yes. Would you prefer I strip it out?

Reply:
Maybe mark with an explicit TODO note, or "TODO (fureigh)"?